packages/gestalt/src/Text.js

```diff
@@ -111,12 +117,13 @@ const TextWithForwardRef: AbstractComponent<Props, HTMLElement> = forwardRef<Pro
       align === 'forceLeft' && typography.alignForceLeft,
       align === 'forceRight' && typography.alignForceRight,
       overflow === 'breakWord' && typography.breakWord,
+      overflow === 'breakAll' && typography.breakAll,
       overflow === 'noWrap' && typography.noWrap,
       italic && typography.fontStyleItalic,
       underline && typography.underline,
       weight === 'bold' && typography.fontWeightSemiBold,
       weight === 'normal' && typography.fontWeightNormal,
-      isNotNullish(lineClamp) && typography.lineClamp,
+      lineClampStyles,
```

Comment:
could we refactor to prevent overrides?

if overflow === 'breakWord' and isLineClamp technically we would be setting breakWord twice
overflow === 'breakWord' && typography.breakWord
&
[typography.breakWord]: isNotNullish(lineClamp) && overflow !== 'breakAll',

Maybe?
`
const isLineClamp = isNotNullish(lineClamp)
...
overflow === 'breakAll' && typography.breakAll,      
{
[typography.breakWord]: overflow === 'breakWord' ||  (isLineClamp && overflow !== 'breakAll')
[typography.lineClamp]: isLineClamp
}
`

or 

...
overflow === 'breakAll' && typography.breakAll, 
(overflow === 'breakWord' ||  (isLineClamp && overflow !== 'breakAll')) && typography.breakWord,
isLineClamp && typography.lineClamp
`

Reply:
I agree, the overrides can be hard to follow. I was thinking through this, but I thought it was super hard to read/think through.

I refactored to a method like this:
```js
 const getOverflowStyle = (): string | void => {
      if (overflow === 'noWrap') {
        return typography.noWrap;
      }

      if (overflow === 'breakAll') {
        return typography.breakAll;
      }

      // default to breakWord if lineClamp is set
      if (overflow === 'breakWord' || isNotNullish(lineClamp)) {
        return typography.breakWord;
      }

      return undefined;
    };
```